cloudencrypt/encryptor.go

```diff
@@ -0,0 +1,24 @@
+package cloudencrypt
+
+import (
+	"context"
+	"crypto/rand"
+
+	"github.com/pkg/errors"
+)
+
+type Encryptor interface {
+	Encrypt(ctx context.Context, value []byte, metadata ...MetadataKV) ([]byte, error)
+	Decrypt(ctx context.Context, encryptedValue []byte, metadata ...MetadataKV) ([]byte, error)
+	Close() error
+}
+
+func randomBytes(size int) ([]byte, error) {
+	bytes := make([]byte, 32)
+	_, err := rand.Read(bytes)
+	if err != nil {
+		return nil, errors.Wrapf(err, "can't generate random bytes: %s", err.Error())
+	}
+
+	return bytes, err
+}
```

Comment:
This won't be used as library if I am right, move it to `internal/utils` or so

Reply:
> This won't be used as library 

What do you mean?

Reply:
You have only things that are used in `cloudencrypt` package. This is some internal function that noone needs to see and typically won't change. Someone can come, change name to uppercase make it public and export it as library internals. Thats why if you move it to internal even though it is public, it is public only for this library

Reply:
I have no clue what you're talking about. This library will be imported using `go.mod` same as any other library.

Reply:
https://github.com/golang-standards/project-layout